Formula/dspdfviewer.rb

```diff
@@ -4,7 +4,7 @@ class Dspdfviewer < Formula
   url "https://github.com/dannyedel/dspdfviewer/archive/v1.15.1.tar.gz"
   sha256 "c5b6f8c93d732e65a27810286d49a4b1c6f777d725e26a207b14f6b792307b03"
   license "GPL-2.0"
```

Comment:
https://github.com/dannyedel/dspdfviewer/blob/555d28b0ec31eb825e939d1b194eaf796d1f8c11/adjustedlink.cpp#L5-L8
```suggestion
  license "GPL-2.0-or-later"
```